site/docs/components/country-symbol/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Country Symbol
+data:
+  description: Country Symbol component displays an icon that represents the flag of a country or inter-governmental organization. You can use Country Symbols individually or as a complete package. The Country Symbols are stylized and representative—so users are more likely to recognize them at smaller sizes, and they maintain their character within the circular form.
```

Comment:
I don't think this is a blocker to this PR, but worth to mention

> a country or inter-governmental organization

This leaves room for political debate for some of the symbols which is not a "country". Given we're using symbol code, it would be worth while to reference some doc on why we chose code, is it ISO?

e.g. ISO 3166 — Codes for the representation of names of countries and their **subdivisions**

Reply:
Yeah might be worth checking with @pseys and tackling in another PR if necessary.

Reply:
Hi, The country codes are aligned to ISO 3166, which defines internationally recognized codes of letters and/or numbers to refer to countries and their subdivisions. The names of countries the codes refer to comes from United Nations sources (Terminology Bulletin Country Names and the Country and Region Codes for Statistical Use maintained by the United Nations Statistics Divisions). The only sub-divisions we have included is a breakdown of the GB countries, although this doesn't limit us from doing more in the future if required. 

I believe the only point of contention is that we have included Kosovo, I think partly because it was already included within the UITK set. Its not officially recognised by the UN but is recognised by 101 out of 193 member states (including the US) and 22 out of 27 EU member states. 

We've followed ISO 3166 and the UN naming conventions to take opinion out of it and to have a source to reference in the future regarding any name changes. 

We've previously had JPM employees question the names we've used because they thought it should match the JPM website, so this was an attempt to remove personal opinion out of the equation and to have something to put to if criticised or questioned in the future.